code/game/turfs/simulated/walls_mineral.dm

```diff
@@ -1,268 +1,293 @@
-/turf/simulated/wall/mineral

```

Comment:
Actual changes are from line 223-248. I did a bad and improperly deleted the file prior to re-submission.